frontend/src/app/modules/home/organization/create-organization/create-organization.component.html

```diff
@@ -0,0 +1,58 @@
+<p-dialog [(visible)]="display" appendTo="body" [modal]="true" styleClass="new-org">
+    <ng-template pTemplate="header">
+        <span class="header">Create organization</span>
+    </ng-template>
+    <form [formGroup]="formGroup" method="POST">
+        <div class="reg-block">
+            <div class="input-block">
+                <span class="p-float-label">
+                    <input id="float-input-comp-name" type="text" formControlName="organizationName" name="comp-name"
+                        pInputText>
+                    <label for="float-input-comp-name">Organization name</label>
+                </span>
+                <div *ngIf="formGroup.controls.organizationName.touched && formGroup.controls.organizationName.invalid"
+                    class="validation">
+                    <small *ngIf="formGroup.controls.organizationName.errors.required">Сan not be empty</small>
+                    <small *ngIf="formGroup.controls.organizationName.value?.length < 3">Min length 3</small>
+                    <small *ngIf="formGroup.controls.organizationName.value?.length > 50">Max length 50</small>
+                    <small *ngIf="formGroup.controls.organizationName.hasError('pattern')">May contain letters, numbers,
```

Comment:
Maybe it's better to use get/set in ts file instead of writing this pieces of code

Reply:
Fixed